cypress/e2e/registration-listspec.cy.js

```diff
@@ -0,0 +1,32 @@
+describe('empty spec', () => {
+  it('passes', () => {
+    cy.visit('http://localhost:3000/registration-list')
+  })
+})
+
+describe('registration list page', () => {
+  it('displays the title', () => {
+    cy
+    .visit('http://localhost:3000/registration-list')
```

Comment:
nit: To reduce repetition, you could put the `visit` in a `before` hook (https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Hooks)